.github/workflows/ci.yml

```diff
@@ -52,30 +52,28 @@ jobs:
       - name: Lint
         run: make lint
 
-
-  generated-files-integrity:
-    runs-on: ubuntu-20.04
+  buf-build:
+    runs-on: ubuntu-latest
     steps:
-      - name: Install Go
+      - name: checkout
+        if: success()
+        uses: actions/checkout@v2
+        with:
+          ref: master
+      - name: checkout-master
```

Comment:
Why we need checkout master for 3 times?

Reply:
This needs to be fixed, not sure why buf recommended it.